xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/notification/macros.vm

```diff
@@ -110,6 +110,12 @@
       #end
     </div>
     <div class="col-xs-9 notification-content">
+      <button class="notification-event-read-button btn btn-xs hidden"
+              aria-label="$services.localization.render('notifications.macro.markEventAsRead')"
+              title="$services.localization.render('notifications.macro.markEventAsRead')"
```

Comment:
Shouldn't those be xml escaped?

Reply:
Maybe, here I reuse same structure as lines below (for the details button).

Reply:
Definitely, this needs escaping, imagine a translation `Mark this "event" as read`. Existing uses aren't good as examples, we have even translations that contain XML-escaping... Note that previously, escaping was automatic as JavaScript APIs were used for setting the attributes.